SPECS/libXaw/libXaw.spec

```diff
@@ -54,23 +54,26 @@ export CFLAGS="%{optflags} -Os"
 make %{?_smp_mflags}
 
 %install
-
 make install DESTDIR=%{buildroot} INSTALL="install -p"
+install -pm 644 COPYING README.md ChangeLog %{buildroot}/%{_pkgdocdir}
 find %{buildroot} -type f -name "*.la" -delete -print
 
 %post -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
 
 %files
 %license COPYING
-%doc ChangeLog README
+%doc ChangeLog README.md
+%dir %{_pkgdocdir}
+%{_pkgdocdir}/ChangeLog
+%{_pkgdocdir}/COPYING
+%{_pkgdocdir}/README.md
 %{_libdir}/libXaw.so.7
 %{_libdir}/libXaw7.so.7
 %{_libdir}/libXaw7.so.7.0.0
 
 %files devel
 %dir %{_includedir}/X11/Xaw
-%dir %{_pkgdocdir}
 %{_includedir}/X11/Xaw/*.h
 # FIXME: Is this C file really supposed to be here?
```

Comment:
Could we check on this FIXME? If it's supposed to be there, can we remove this comment?

Reply:
this is issue is still present